core/pom.xml

```diff
@@ -394,6 +394,8 @@
           <argLine>
             @{jacocoArgLine}
             -Djava.library.path=${project.build.directory}/hyperic-sigar-${sigar.base.version}/sigar-bin/lib/
+            -Duser.language=en
+            -Duser.country=US
```

Comment:
@viongpanzi I tested this locally by setting different country / languages. It appears that we only need the language set to en to get the TimestampParserTest to parse. I don't think you need to set the country - can you confirm?

Also, I think the issue here isn't tied to whether or not the timezone is in UTC or not. It's an issue with the timestamp format used in the tests not being supported by all locales.

Reply:
@suneet-s Yes, you're right! We don't need to set the country. Patch and the title are updated!